trio/_core/_tests/test_multierror_scripts/apport_excepthook.py

```diff
@@ -12,4 +12,4 @@
 
 import trio
 
-raise trio.MultiError([KeyError("key_error"), ValueError("value_error")])
+raise trio.MultiError([KeyError("key_error"), ValueError("value_error")])  # type: ignore[attr-defined]
```

Comment:
For this and the similar thing in some similar tests, I am curious what the error is in particular. If you could add a comment after the ignore about why, that would be handy.

Reply:
Since `MultiError` is deprecated, we hid it from type checkers to make them error out on any code that uses it.